src/assets/common/index.ts

```
@@ -1,3 +1,5 @@
 export { default as Background } from './background.webp';
 export { default as Logo } from './logo.webp';
 export * from './svgs';
+export { default as HomeLogo } from './home-logo.webp';
+export { default as Shelve } from './empty-shelve.webp';
```

ESLint

[error] Run autofix to sort these exports!

(simple-import-sort/exports)